database/knex.js

```diff
@@ -0,0 +1,38 @@
+require('dotenv').config()
+const knex = require('knex')(knexConfig);
+const expect = require('expect-runtime');
+const connection = require('../config/config').connectionString;
+
+expect(connection).to.match(/^postgresql:\//);
+const log = require("loglevel");
+
+const knexConfig = {
+  client: 'pg',
+  debug: process.env.NODE_LOG_LEVEL === "debug",
+  connection,
+  pool: { 
+    min:0, 
+    max: 100,
+    afterCreate (conn, done) {
+      conn.query('SET SESSION CHARACTERISTICS AS TRANSACTION ISOLATION LEVEL SERIALIZABLE;', function (err) {
```

Comment:
This level of transaction isolation is not necessary for this API and will introduce unnecessary complexity for this use case.

Reply:
Removed the afterCreate clause